plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -583,6 +613,12 @@ private void invalidateColumnsCache(SchemaTableName table)
         invalidateCache(columnsCache, key -> key.table.equals(table));
     }
 
+    @VisibleForTesting
```

Comment:
Can we have this change as a separate commit ? As it allows Trino to expose schema stats ?

Reply:
And add some test for schema stats ?